src/Germany/Mews.Fiscalizations.Germany/V2/RequestCreator.cs

```diff
@@ -152,5 +152,10 @@ private static Dto.ClientState SerializeClientState(ClientState state)
                 ClientState.Deregistered, _ => Dto.ClientState.DEREGISTERED
             );
         }
+
+        private static string ToString(decimal amount)
+        {
+            return String.Format(CultureInfo.InvariantCulture, "{0:F2}", amount);
```

Comment:
Sending an amount with more than 2-5 decimals would violate fiskaly scheme (validation). so we should round the decimal.

Reply:
So why not round to 5 then? Why 2?